src/toxcore/dht/packet/mod.rs

```diff
@@ -15,7 +15,7 @@ mod cookie_response;
 mod bootstrap_info;
 mod lan_discovery;
 mod crypto_handshake;
-mod crypto_data;
+pub mod crypto_data;
```

Comment:
No need to make it public - we have `pub use`.

Reply:
>use crate::toxcore::dht::packet::crypto_data::MAX_CRYPTO_DATA_SIZE;

is used.

Reply:
Have a look at `pub(crate)`

Reply:
Used ....::packet{Packet as DhtPacket, *};